### PR TITLE
Create database seeder

### PR DESCRIPTION
**Pull request by NathanABrooks**
We need to create a database seeder so our front end developers have some dummy data to work with when deciding how the interfaces will look and interact.

### CODE REVIEW

**Comment by NathanABrooks** [comment]
THIS IS **NOT** READY TO MERGE PLEASE DO NOT MAKE CHANGES WITHOUT LETTING ME KNOW
There is still a couple of bugs to iron out, this should be ready by the end of Thursday to check over.

**Comment by NathanABrooks** [comment]
While there are still issues in terms of efficiency, scalability and usability that could be addressed,
I can't afford to spend any more time on this right now.

Despite this, all the bugs are ironed out and this ***should*** provide the front end developers with plenty of data to use.

**Review by ThomasJameson — APPROVED** [review]
Looks all good, logical as expected.

File: database/factories/sessionFactory.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+| This directory should contain each of the model factory definitions for
+| your application. Factories provide a convenient way to generate new
+| model instances for testing / seeding your application's database.
+|
+*/
+
+$factory->define(Session::class, function (Faker $faker) {
+    // Ensuring that the ending dates are always a reasonable time after the starting dates.
+    $startingTime = $faker->optional($weight = 0.8)->dateTimeThisYear('+1 month');
+    if ($startingTime) {
```
Comment (ThomasJameson) [inline database/factories/sessionFactory.php:24]: Should be consistent with previous variable names, 'startTime' rather than 'startingTime'

**Review by ghost — APPROVED** [review]

**Pull request merged**